app/src/main/java/com/qudus/tudee/designSystem/component/TabBar.kt

```diff
@@ -0,0 +1,119 @@
+@Composable
+fun TabBar(
+    startTab: TaskStatus = TaskStatus.IN_PROGRESS,
+    onTabSelected: (TaskStatus) -> Unit = {},
+) {
+    var selectedTabIndex by rememberSaveable { mutableStateOf(startTab.ordinal) }
+    val tabs = TaskStatus.values()
+
+
+    PrimaryTabRow(
+        selectedTabIndex = selectedTabIndex,
+        containerColor = Theme.color.surfaceHigh,
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(48.dp),
+        divider = {
+            Box(
+                Modifier
+                    .fillMaxWidth()
+                    .height(1.dp)
+                    .background(Theme.color.stroke)
+            )
+        },
+        indicator = {
+            TabRowDefaults.PrimaryIndicator(
+                modifier = Modifier
+                    .tabIndicatorOffset(selectedTabIndex, matchContentSize = true),
+                height = 4.dp,
+                width = Dp.Unspecified,
+                color = Theme.color.secondary,
+                shape = RoundedCornerShape(topStart = 12.dp, topEnd = 12.dp)
+            )
+        }
+    ) {
+        tabs.forEachIndexed { index, tab ->
+
+            Tab(
+                selected = selectedTabIndex == index,
+                onClick = {
+                    selectedTabIndex = index
+                    onTabSelected(tab)
+                },
+                selectedContentColor = Theme.color.title,
+                unselectedContentColor = Theme.color.hint,
+                text = {
+                    Row(
+                        verticalAlignment = Alignment.CenterVertically,
+                    ) {
+                        Text(
+                            text = tab.title,
+                            textAlign = TextAlign.Center,
+                            style = if (selectedTabIndex == index)
+                                Theme.textStyle.label.medium else Theme.textStyle.label.small,
+                        )
+
+                        if (tab.count > 0 && selectedTabIndex == index) {
+                            Box(
+                                modifier = Modifier
+                                    .padding(start = 4.dp)
+                                    .size(26.dp)
+                                    .background(Theme.color.surface, CircleShape),
+                                contentAlignment = Alignment.Center
+                            ) {
+                                Text(
+                                    text = tab.count.toString(),
+                                    style = Theme.textStyle.label.medium,
+                                    color = Theme.color.body
+                                )
+                            }
+                        }
+                    }
+                }
+            )
+        }
+
+    }
+}
+
+enum class TaskStatus(val title: String, val count: Int) {
+    IN_PROGRESS("In progress", 14),
+    TODO("To Do", 2),
+    DONE("Done", 3)
```

Comment:
Depend what you give number 14,2,3 this hard code you can just take it from parameter